docs/user-guide/advanced/Pandas_API.ipynb

```diff
@@ -2259,6 +2259,68 @@
     "tab.abs(numeric_only=True)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "0c056fd9-fe7b-43d5-b1c7-7ceec3cae5ff",
+   "metadata": {},
+   "source": [
+    "### Table.round()\n",
+    "\n",
+    "```\n",
+    "Table.round(self, decimals: Union[int, Dict[str, int]] = 0)\n",
+    "```\n",
+    "\n",
+    "Round a Table to a variable number of decimal places.\n",
+    "\n",
+    "**Parameters:**\n",
+    "\n",
+    "| Name             | Type                | Description                                                   | Default |\n",
+    "| :--------------: | :-----------------: | :------------------------------------------------------------ | :-----: |\n",
+    "| decimals         | int or Dict | Number of decimal places to round each column to. If an int is given, round each real or float column to the same number of places. Otherwise dict rounds to variable numbers of places. Column names should be in the keys if decimals is a dict-like and the decimals to round should be the value. Any columns not included in decimals will be left as is. Elements of decimals which are not columns of the input will be ignored.| 0       |\n",
+    "\n",
+    "**Returns:**\n",
+    "\n",
+    "| Type       | Description                                                                              |\n",
+    "| :--------: | :--------------------------------------------------------------------------------------- |\n",
+    "| Table      | A Table with the affected columns rounded to the specified number of decimal places. |"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1b629def",
+   "metadata": {},
+   "source": [
+    "If an integer is provided it rounds every float column to set decimals."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "08c182c9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tab.round(1)"
```

Comment:
"I think it would be good to build an example table to see how it works.

Reply:
The table is defined in [line 2220](https://github.com/hablapps/pykx/blob/17b90247f252bd924ec099279a8e8920dfa20453/docs/user-guide/advanced/Pandas_API.ipynb?short_path=9712990#L2220). Since this function is part of the "Computations" chapter, I thought it would be better to reuse the existing table, since it reduces the noise a little bit.

Reply:
Very well, I prefer the manner in which you handle it.